fast_api_als/routers/submit_lead.py

```diff
@@ -37,13 +37,15 @@ async def submit(file: Request, apikey: APIKey = Depends(get_api_key)):
     t1 = [int(time.time() * 1000.0)]
```

Comment:
too many useless info logs in this file, exception handling missing at 2-3 key points